dist/bin/dotd

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
+  ## resolve links - $0 may be a link to PROG_HOME
+  PRG="$0"
+
+  # need this for relative symlinks
+  while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
+    link=`expr "$ls" : '.*-> \(.*\)$'`
+    if expr "$link" : '/.*' > /dev/null; then
+      PRG="$link"
+    else
+      PRG="`dirname "$PRG"`/$link"
+    fi
+  done
+
+  saveddir=`pwd`
+
+  PROG_HOME=`dirname "$PRG"`/..
+
+  # make it fully qualified
+  PROG_HOME=`cd "$PROG_HOME" && pwd`
+
+  cd "$saveddir"
+fi
+
+source "$PROG_HOME/bin/common"
+
+default_java_opts="-Xmx768m -Xms768m"
+bootcp=true
+
+PROG_NAME=dotty.tools.dottydoc.Main
```

Comment:
@felixmulder We've a long list of dependencies if `dottydoc` is included. It doesn't look very nice. How about we do some `assembly` for `dottydoc` and do it in a separate PR?

1. ST4-4.0.7.jar
1. antlr-3.5.1.jar
1. antlr-runtime-3.5.1.jar
1. autolink-0.6.0.jar
1. flexmark-0.11.1.jar
1. flexmark-ext-anchorlink-0.11.1.jar
1. flexmark-ext-autolink-0.11.1.jar
1. flexmark-ext-emoji-0.11.1.jar
1. flexmark-ext-gfm-strikethrough-0.11.1.jar
1. flexmark-ext-gfm-tables-0.11.1.jar
1. flexmark-ext-gfm-tasklist-0.11.1.jar
1. flexmark-ext-ins-0.11.1.jar
1. flexmark-ext-superscript-0.11.1.jar
1. flexmark-ext-tables-0.11.1.jar
1. flexmark-ext-wikilink-0.11.1.jar
1. flexmark-ext-yaml-front-matter-0.11.1.jar
1. flexmark-jira-converter-0.11.1.jar
1. flexmark-util-0.11.1.jar
1. jackson-annotations-2.2.3.jar
1. jackson-core-2.8.6.jar
1. jackson-databind-2.2.3.jar
1. jackson-dataformat-yaml-2.8.6.jar
1. jsoup-1.7.2.jar
1. liqp-0.6.7.jar
1. snakeyaml-1.17.jar